manim/mobject/svg/code_mobject.py

```diff
@@ -94,8 +95,8 @@ def construct(self):
         Number of space characters corresponding to a tab character. Defaults to 3.
     line_spacing : :class:`float`, optional
         Amount of space between lines in relation to font size. Defaults to 0.3, which means 30% of font size.
-    scale_factor : class:`float`, optional
-        A number which scales displayed code. Defaults to 0.5.
+    font_size : class:`float`, optional
+        A number which scales displayed code. Defaults to 24.
```

Comment:
```suggestion
        A number which scales displayed code. Defaults to 24. To convert old scale factors to font size, multiply by 48.
```

Reply:
It's a little akward to put this in the docs.

Reply:
Sure.